engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/pdf/font/FontMappingConfig.java

```diff
@@ -21,41 +21,65 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * Font mapping configuration class
+ *
+ * @since 3.3
+ *
+ */
 public class FontMappingConfig {
 
-	protected Set fontPaths = new HashSet();
+	protected Set<String> fontPaths = new HashSet<String>();
 
 	/** The font-family replacement */
-	protected HashMap fontAliases = new HashMap();
+	protected HashMap<String, String> fontAliases = new HashMap<String, String>();
 
 	/** The encoding for the fonts */
-	protected HashMap fontEncodings = new HashMap();
+	protected HashMap<String, String> fontEncodings = new HashMap<String, String>();
 
 	/** the global sequences defined for composite fonts */
-	protected HashMap searchSequences = new HashMap();
+	protected HashMap<String, String[]> searchSequences = new HashMap<String, String[]>();
 
 	/**
 	 * composite fonts is constructed by multiple physical fonts which may cover
 	 * large amount of glyph
 	 */
-	protected HashMap compositeFonts = new HashMap();
+	protected HashMap<String, CompositeFontConfig> compositeFonts = new HashMap<String, CompositeFontConfig>();
 
+	/**
+	 * Constructor
+	 */
 	public FontMappingConfig() {
 	}
 
+	/**
+	 * merge the font configuration to the existing font setup
+	 *
+	 * @param config font mapping configuration
+	 */
 	public void merge(FontMappingConfig config) {
 		fontPaths.addAll(config.fontPaths);
-		fontAliases.putAll(config.fontAliases);
+
+		// fontAliases.putAll(config.fontAliases);
```

Comment:
Remove this, we have the old behavior in the GIT history.

Reply:
Nor problem, I will remove it.